nrfutil/device/logLibVersions.ts

```diff
@@ -108,15 +108,19 @@ export default async (moduleVersion: ModuleVersion) => {
                     `It looks like you have installed JLink using ${JLinkInstallerVersion}, but currently we only support their Universal Installer for your system.`
                 );
                 logger?.warn(
-                    `Please install JLink: https://www.segger.com/downloads/jlink/JLink_MacOSX_V788j_universal.pkg`
+                    `Please install JLink: https://www.segger.com/downloads/jlink/JLink_MacOSX_V794e_universal.pkg`
                 );
             }
         }
     } catch (error) {
-        logger?.error(
-            `Failed to get the library versions${
-                error == null ? `: ${describeError(error)}` : ''
-            }`
-        );
+        let message = '';
+        if (error == null) {
+            message = describeError(error);
+        }
+        if (error === undefined) {
+            message = '';
+        }
```

Comment:
This is not the same as 

`error == null ? `: ${describeError(error)}` : ''`

error == null is true for null and undefined as we are using == and not ===

regardless I don't understand why we use describeError(error) when error is null or undefined IMO it should be 

`logger?.error(
             `Failed to get the library versions${
                 error != null ? `: ${describeError(error)}` : ''
             }`
         );`